README.md

```diff
@@ -95,6 +95,37 @@ The benchmark assumes that Vivaldi uses bottom navigation, available in settings
 
 Should work out of the box.
 
+#### OEM "optimizations"
+
+In the past OEMs have been caught artifically limiting the performance of certain detected Apps which often included various different browsers. Notably Oneplus ([source](https://arstechnica.com/gadgets/2021/07/oneplus-admits-to-throttling-phones-after-launch-to-improve-battery-life/#:~:text=We%20have%20detected%20that%20OnePlus%20is%20blacklisting%20popular,top%20popular%20non-benchmark%20apps%20get%20notably%20reduced%20performance.)) and Samsung ([source](https://www.androidauthority.com/samsung-gos-throttling-apps-3125885/)) in the last couple of years.
+Since the purpose of the benchmark is not to test some OEM cheating software you should perform a quick sanity check if the performance of your browser roughly falls roughly in line with other devices with the same chipset or primary CPU core. 
+The benchmark [Octane JS by Google](https://chromium.github.io/octane) is a pure Javascript benchmark and runs in less than a minute and yields very constistant results. If the results for your device are more than 30% off its likley that some manipulation is going on. You can also try to prevent this by enabling any performance mode and disabling any battery saving related features on the device if available, however in the case of Samsung and Oneplus this still did not remove the artifical limits.
```

Comment:
```suggestion
If you'd like to test whether performance throttling may be occurring, try running the [Octane JS benchmark by Google](https://chromium.github.io/octane) - it runs in less than a minute and yields very consistent results. Compare to the table below - if the results for your device/SoC are more than 30% off, it's likely that some manipulation is going on. You can try to prevent this by disabling battery saving features or enabling "performance modes" - however, this may still not remove the artificial limits on Samsung and OnePlus devices.
```